app/test-coordinates/page.tsx

```
@@ -0,0 +1,358 @@
+    setSelectedAreas([])
+    setActiveTab('upload')
+    
+    // 테스트 상태 초기화
+    setTestResults(prev => prev.map(result => ({ ...result, status: 'pending' })))
+    setCurrentTest('')
+    
+    console.log('🗑️ 이미지 클리어 완료')
+  }
+
+  // 선택된 영역 변경 처리
+  const handleAreasChange = (areas: any[]) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)